draft-ietf-taps-transport-security.md

```diff
@@ -496,14 +496,17 @@ using EAP for example.
   - tcpcrypt
 
 - Pre-Shared Key Import:
-Either the handshake protocol or the application directly can supply pre-shared keys for the
-record protocol use for encryption/decryption and authentication. If the application can supply
-keys directly, this is considered explicit import; if the handshake protocol traditionally
-provides the keys directly, it is considered direct import; if the keys can only be shared by
-the handshake, they are considered non-importable.
-  - Explicit import: QUIC, ESP, OpenVPN
-  - Direct import: TLS, DTLS, tcpcrypt, MinimalT, WireGuard
-  - Non-importable: CurveCP
+Either the handshake protocol or the application directly can supply pre-shared keys for use
+in encrypting (and authenticating) connections to a peer.
```

Comment:
Maybe let's substitute "connections" here with "communications" or "data sent"? Not all of these protocols are connection-oriented.

Reply:
```suggestion
in encrypting (and authenticating) communication with a peer.
```